# core/errors.py
class ConfigurationError(Exception):
    pass